<launch>

  <include file="$(find openni_launch)/launch/openni.launch">
    <arg name="load_driver" value="true" />
    <arg name="depth_registration" value="true" />
  </include>

  <node pkg="suturo_perception_rosnode" name="suturo_perception_rosnode" type="suturo_perception_rosnode" output="screen"></node>
  <param name="suturo_perception/point_topic" type="string" value="/camera/depth_registered/points" />
  <param name="suturo_perception/frame_id" type="string" value="camera_rgb_optical_frame" />

</launch>
